### ISSUE TITLE
Feature request: more freedom when placing multiple parts

### ISSUE THREAD

**Issue by m-vo** [daid/LegacyCura#1110]
Sorry for spamming the issue tracker today - this one is quite low prio. :-)

Example: When placing an U-shaped model Cura does not allow to place other objects inside the U. It seems like one cannot place anything inside the convex hull of another object. > I assume that is for easier collision detection (and other implications like travel paths), but would on the other hand allow denser packaging of objects on the buildplate. Perhaps this might at least be possible if the user selects the "all at once" option.

**Comment by thekroko** [comment]
Agreed that this would be a really nice feature. I feel like the detection should either 
(a) use the skirt instead of the hull (not sure how complex intersection detection detection is in this case), 
or (b) there should be an option to (temporarily) turn off the auto-alignment for overlapping objects so that the objects can be placed close together manually. Right now it is wasting a lot of bed space.
